### PR TITLE
Sort imports for analyzer, admin and agent folders

### PR DESCRIPTION
**Pull request by TheMythologist**
- Added pyproject.toml to standardise isort configuration
- Removed unused imports
  - Imports that are used in comments are commented
- Standardisation: 1 newline / start of file before start of imports

### CODE REVIEW

File: analyzer/linux/lib/api/screenshot.py [hunk 1]
```diff
@@ -1,24 +1,14 @@
...
 
 log = logging.getLogger(__name__)
 
 try:
-    log.debug("Importing 'pyscreenshot'")
-    try:
-        import pyscreenshot as ImageGrab
-    except ImportError:
-        log.error("Missed dependency: pip3 install pyscreenshot")
-        HAVE_PIL = False
```
Comment (TheMythologist) [inline analyzer/linux/lib/api/screenshot.py:21]: `pyscreenshot` was removed as ImageGrab would simply be overwritten by `from PIL import ImageGrab`.
In cases where `from PIL import ImageGrab` fails, `HAVE_PIL` would be False and `ImageGrab` would not be called, so there is no difference

**Comment by TheMythologist** [comment]
Feel free to edit `pyproject.toml` as you see fit

File: analyzer/linux/lib/api/screenshot.py (hunk 1 above)
Comment (doomedraven) [reply analyzer/linux/lib/api/screenshot.py:21]: ok thanks for clarification

**Comment by doomedraven** [comment]
thanks wanted to add `isort` for long time but never had time to start with

**Pull request merged**